packages/components/src/components/popover/popover-styles.ts

```diff
@@ -209,12 +209,23 @@ export const getComponentCss = (direction: PopoverDirection, theme: Theme): stri
       transform: 'translate3d(0,0,0)', // Fixes movement on hover in Safari
     },
     spacer: {
-      position: 'absolute',
+      ...(isNative
+        ? {
+            position: 'absolute',
+            overflow: 'initial',
+            backgroundColor: 'transparent',
+            border: 'none',
+            margin: 0,
+            padding: 0,
```

Comment:
Do we really need these styles? Looks like they try to reset something but where is it coming from?

Reply:
Necessary to overwrite the native popover styles